README.md

```diff
@@ -9,8 +9,9 @@ Python CLI app to detect if a screenshot of a tweet is genuine. Work in Progress
 ### Setting Up
 
 * Run `python setup.py install` to install requirements. Install script is still a work in progress.
-* Run `python fakemenot/app.py` after installation.
-* Go to https://apps.twitter.com/app/new and put your consumer and API keys in `fakemenot/twitter.config`
+* Copy `fakemenot/twitter.config` to ~/.fakemenot.config
```

Comment:
Good thinking about the naming. Don't you think it will be better to add the `cp fakemenot/twitter.config ~/.fakemenot.config` command so that people can follow along?

Reply:
Sure